src/devices/CharacterLcd/IGpioController.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.CharacterLcd
+{
+    // Can we move this upstream and actually derive from this? Should I2C/SPI derive from GpioDriver instead?
+    // Having an interface would allow for easier unit testing...
+    public interface IGpioController : IDisposable
+    {
+        void OpenPin(int pinNumber, PinMode mode);
+        void ClosePin(int pinNumber);
+        void SetPinMode(int pinNumber, PinMode mode);
+        void Write(int pinNumber, PinValue value);
```

Comment:
I thought you planned to include a Write overload that takes in ValueTuple<int, int> so that we can support writing to several pins at the same time?

Reply:
Yes, in a PR after this one gets submitted. Too big for this one.

Reply:
I included the change for this now.